PostReleaseActivities.md

```diff
@@ -16,6 +16,8 @@ Please follow the below steps after publishing analyzer NuGet packages from this
       1. Bump up the `VersionPrefix`. If the new version prefix is greater then or equals the current `AnalyzerUtilitiesVersionPrefix`, then update `AnalyzerUtilitiesVersionPrefix` to `$(VersionPrefix)`.
       2. Reset `PreReleaseVersionLabel` to `beta1`.
       3. Update `DogfoodNetAnalyzersVersion` and/or `DogfoodAnalyzersVersion` to the latest released package version.
+      4. For `Microsoft.CodeAnalysis.Analyzers` package, update [`MicrosoftCodeAnalysisAnalyzersVersion`](https://github.com/dotnet/roslyn/blob/95809b0b922439465a213922ef7eb81e9b5a223f/eng/Versions.props#L82) in dotnet/roslyn to reference the new package version.
```

Comment:
This seems odd to put steps to be done in a separate repo over here, all the existing steps are for this repo. Instead it would be better to just create a new section at the end of this page for these 2 new follow up items.